### DISCUSSION TITLE
qemu: uncaught target signal 11 (Segmentation fault) - core dumped

### DISCUSSION THREAD

**Discussion by djcrawleravp** [containrrr/watchtower#1155] [Q&A] [answered]
After rebooting my Raspberry pi I've got this message

qemu: uncaught target signal 11 (Segmentation fault) - core dumped

My docker compose for watchtower is

  watchtower:
    container_name: Watchtower
    image: containrrr/watchtower:latest
    network_mode: bridge
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock
    restart: unless-stopped
    environment:
      - WATCHTOWER_SCHEDULE=0 0 5 * * *
      - TZ=America/La_Paz
      - WATCHTOWER_CLEANUP=true
      - WATCHTOWER_INCLUDE_STOPPED=true
      - WATCHTOWER_REMOVE_VOLUMES=true
      - PUID=1001
      - PGID=1001
      
      
Been receiving messages reading that maybe I need docker login to update my containers in the past weeks

**Comment by piksel** [comment, marked as answer]
Where do you see this message? If that is in the watchtower logs it would suggest you are somehow running the wrong image (for another platform).
